docs/languages/cpp.md

```diff
@@ -290,10 +266,6 @@ Read on to find out about:
 
 ## Common Questions
 
-**Q: I can't debug multi-threaded code.**
-
-**A:** For multi-threaded debugging, you must use the version 1.1.1 of the C/C++ Extension for Visual Studio Code.
-
 **Q: My project won't load.**
 
 **A:** VS Code doesn't currently support C++ project files, instead it considers a directory of your choosing to be the workspace of your project. Source code files inside that directory and its sub-directories are part of the workspace.

```

Comment:
Please add a section like:
Q: Where do I report issues or get help?
A: Please file an issue one [here] the GitHub link